docs/image-sizes.md

```diff
@@ -0,0 +1,95 @@
+# Defining Image Sizes
+
+Altis provides some default image sizes, that you can use to render images in the theme.
+
+- Thumbnail
+- Medium
+- Large
+- Post Thumbnail
+- Full
+
+They have default dimensions, but you can override these either with code or in the Admin, under Settings > Media.
+
+![Media Settings](https://user-images.githubusercontent.com/30460/67079140-8b846000-f18a-11e9-8387-038b594f19aa.png)
+
+A lot of times, you'll find that you need additional image sizes for different contexts, depending on your theme's design.
+
+You can define additional image sizes with  the`add_image_size` function.
+
+This function accepts the following parameters:
+
+```php
+ * @param string     $name   Image size identifier.
+ * @param int        $width  Optional. Image width in pixels. Default 0.
+ * @param int        $height Optional. Image height in pixels. Default 0.
+ * @param bool|array $crop   Optional. Whether to crop images to specified width and height or resize.
+```
+The custom image sizes should be declared in the callback function to the `after_setup_theme` action.
+
+Example:
+```php
+add_action( 'after_setup_theme', __NAMESPACE__ . '\\theme_setup' );
+function theme_setup() {
+	// Set the image size by resizing the image proportionally (without distorting it):
+	add_image_size( 'custom-size', 220, 180 ); // 220 pixels wide by 180 pixels tall, soft proportional crop mode
```

Comment:
```suggestion
	add_image_size( 'custom-size', 220, 180 ); // Image constrained to 220 pixels wide by 180 pixels tall
```
My feeling here is that we're not actually doing any cropping in this instance so "soft crop" doesn't really say anything.